i18n.schemas.js

```diff
@@ -1,14 +1,17 @@
-/* eslint-disable */
 const en = {
-  "lifelines-webshop-sidebar-header": "Select cohorts",
-  "lifelines-webshop-content-header": "Select data",
-  "lifelines-webshop-signed-out-content-header": "View data",
-  "lifelines-webshop-cart-header": "Order variables",
-  "ui-form:form_file_change": "Change",
-  "ui-form:form_file_browse": "Browse",
+  'lifelines-webshop-sidebar-header': 'Select cohorts',
```

Comment:
needs to be added to the database setup script as well

Reply:
Done